sdk/python/feast/infra/aws.py

```diff
@@ -38,11 +39,14 @@
 
     raise FeastExtrasDependencyImportError("aws", str(e))
 
+_logger = logging.getLogger(__name__)
+_logger.setLevel(logging.INFO)
```

Comment:
should log levels be set here? I think logging should be globally configured right?